### PR TITLE
added shadow_upload.sh

### PR DESCRIPTION
**Pull request by ToivoHn**
added shadow_upload.sh

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -245,3 +246,14 @@ Presuming we do not want the driver to auto-launch for our example, the final co
...
 ```
+
+### Customer Key ```-ck```
+
+For uploading ROS_LOGS folder (including .bag file and core dump) to AWS, the user running the oneliner needs to enter a customer key using the -ck flag in the oneliner. To get a customer key, please contact devops@shadowrobot.com. To set an AWS customer key, go to: https://eu-west-2.console.aws.amazon.com/apigateway/home?region=eu-west-2#/api-keys/sf0rfsfv38 (Shadow AWS account needed). By default, -ck flag wont be used, which means ROS LOGS will not be uploaded to AWS. For example, adding ```-ck``` to the oneliner command will prompt you for the customer key, save the customer key inside the user's Docker container and enable ROS_Logs_Saver to upload the whole folder of saved logs to AWS
+ 
+Presuming we do not want the driver to auto-launch for our example, the final command would be:
+
+```bash
+bash <(curl -Ls http://bit.do/launch-sh) -i shadowrobot/flexible-hand:kinetic-v0.2.28 -n flexible_hand_real_hw -e enp0s25 -r true -g false -l false -ck
```
Comment (biofotis) [inline docker/README.md]: i think that needs to be -ck true

Comment (biofotis) [reply docker/README.md]: also in the description above

File: docker/utils/s3_upload_email_lambda_function.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import json
```
Comment (biofotis) [inline docker/utils/s3_upload_email_lambda_function.py]: You should put license information at the top like the other python files.

Comment (ToivoHn) [reply docker/utils/s3_upload_email_lambda_function.py]: done

**Comment by ToivoHn** [comment]
I have make the requested changes

**Pull request merged**